### ISSUE TITLE
Gun Computer > Gun Heist (Rework)

### ISSUE THREAD

**Issue by ImArteria** [InZidiuZ/op-framework-issue-tracker#2524]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?
Yes

**Summary:**
Hey guys, I'm here submitting this idea with one of the biggest problems in the criminal side of things at the moment. Guns and the Gun Computer. Now we're also presented with this problem.
- Gun Computers at the moment can be abused, and people do whatever they want with them. 
- Granted eventually we will update to a new script, but what can we do to keep things fresh and fun for everyone?

I present, the Gun Computer Heist. The Gun Computer at the moment is a "Low Key Process", triggering 10-66 calls every once in a blue moon, and ran at odd times without PD RP. So I present the following idea. 
Lets not have the gun computer be a low key process. Let's make it a high key robbery.

So the robbery would be here
https://i.imgur.com/17v0CWx.jpeg

In GTA 5 lore, this is the Merryweather building. Merryweather are the people with the big guns, and etc. So why not rob them in RP?
Essentially we take the gun computer script, place the gun computer here.
https://i.imgur.com/ExMwAjN.jpeg

This is a building inside the dock full of computers and whatnot already, so no props would need to be placed. Now the hack would commence. After all of the hacking, the crates are spawned. Yes, I said crate(s).  The number would need to be adjusted, but perhaps like 7-15 crates could spawn, being full of random weapons. You could be really lucky, or really not lucky. Due to that amount of weapons, you don't want to go into this robbery by yourself or even just one friend.

**Crate Spawn**
The crates would spawn on this mini platform, at the end of the docks, scattered out.
https://i.imgur.com/PDWXEww.jpeg

At this point is where some development changes would have to happen. 
- Keep in mind, running the computer only requires 1 of each key color. Perhaps adding a new item or something along those lines, so it isn't the easiest heist. 
- Change the item required to open a crate thermite, so a lot of it would be needed depending on the crates.

At this point would depend on the new PD SOPs for this robbery. Does it instantly go to a shootout because you're importing Class 3 Weapons? Or is it just like another chase? 

**Reason:**
Why would your suggestion be a good change to the framework?
At the end of it, I think this robbery would take minimal development and require almost no change. I think it's something new and fresh to bring to the crims, and doesn't come with any problems that the current gun system has. I compiled a pros and cons list.
Below is the dev work required
- Minimum Cop Count (Probably 7-8)
- Long Timer
- Adjustment to how many crates spawn
- Possible new item/hard to get item to open the crate

**Pros**
- Is not a process that can be held down, so you can't camp it.
- Cooldown would be longer so you can't spam it, and would require a minimum cop count. Since it does require that minimum cop count, people can't skip out on RP by doing it at crackhead hours.
- It's a new robbery and huge crim activity, so people would be excited to try it.
- Gun Selling RP can still happen by people who specialize in this sort of activity. 
- Would hold us over until something new comes in.

**Cons**
- Needs some "hard to get" item so dannies can't just meta and rob it

**Comment by KaiEagle** [comment]
I really like the idea but how about making it more harder by moving the location on the military aircraft I know that's really hard but guns ugghh everyone in the city is roaming around with guns currently so it would be fun and hard.

**Comment by ThisIsNotDex** [comment]
Great idea,

In relation to location as Kai stated, I think that would be a terrible spot. Because that would make it holding it down a whole lot easier for criminals, since they can just gun down any helicopter or boat that cops use to get there. And even if not that reason, the criminals get away boats or helicopters could scuff out leaving them stranded.

In regards to the minimum cop count, I say make it 10-13. Because criminals who may want this super high reward, will probably be gang members who can now use 6 people for a process like this. So if there was 7 cops, it already kinda shits on the way cops respond to things where they would typically have 1.5x more people than the crims.

also, preferably make this something that cannot be done at the same time as a bank or jewelry store. Just again for the sake of enough cops to counter since this suggestion is intended to be more role play based.

**Comment by TalosTG** [comment]
I'm gonna be the cop main counter commenter here in regards to the location, and numbers that you stated. 

1. In regards to numbers, every gang on this server for the past 2-3 years has had to make due with 4-6 people at most. The only exception being when something had been designated as their turf. This suggestion ALREADY allows for PD to have superior numbers than criminals. If any gang were offered this opportunity to hijack the gun heist none of us would feel the need to bring 10-13 people, imo the over reliance on far more numbers is what has led to a decrease in strategy and skill in the PD scene lately. 

2. PD in this situation will still have plenty of advantages. In addition to the the increased numbers, remember that every crucial item lost for a crim is far more detrimental than for any officer. Current item cost still means that in the event that a shootout is lost on PD's side the only real consequence is that youre annoyed and lose out on 5 minutes. For a crim you lose out not only on your time, but your money as well. Weapons for us arent something we get to go pick up at a counter, if I spend 17k on a rifle, then get a 10k fine what makes you think im going to feel good about a situation where i had to fight 13 people who only stand to lose 5 minutes and some memory? 

3. The consequences for crims here vs PD alone is enough to justify what imo is a reasonable number count, and location. Realize that if another gang were to hijack the heist, the consequences of this altercation are limited to the gangs in the situation. Those guns are not lost, they will still be put to use by someone. The gun market is what allows conflict RP, and the majority of gang RP to happen. When PD successfully stops a gun run you are stopping every other gang from receiving these weapons. You are actively attacking the market, and with it conflict RP as a whole. 

This is fine if things are balanced out, gangs take into consideration potential losses, buyers understand you wont always win the fights. However a neutral location, with 10-13 police is a situation the PD should NEVER be able to lose (seriously if you lose with more than double the people in a neutral location what are you doing?). Because of this you effectively kill the entire gun market, and completely remove this avenue of RP. We've had situations in the past with events, and processes where most people felt it was not worth it purely because it was unbalanced. 

In conclusion, the location is fine, the numbers are fine, please cop mains learn how to fight without relying on double the people each time. Good suggestion.

**Comment by imdiavolo** [comment]
I'm sorry but I have to agree with Talos here. 
Having this suggestion be the ONLY way to obtain remotely decent guns will simply kill the gun market, I have no doubt in it at all.
Winning a cop shootout where you're heavily disadvantaged in an area where you don't even get to pick is practically impossible. Most if not all wins of crims in shootouts comes from the location they were done at and how good it is to hold down, and it's still pretty fucking hard just because of the horde of PD that you have to face that will slowly wear you out until they eventually do their final push with the zerg of people they somehow still have up even though you downed like 9 of them. 
The risk and reward would be fine if it was an optional thing to do like, if you're a gang who's willing to take that risk even knowing that your chances are next to zero but you care more so about the fun (?) you'll get from it then sure, by all means this is a great idea to have as a SECOND OPTION to the gun computer.

**Comment by enutpeanut** [comment]
I think it would be a cool idea for a high-stakes, high-reward robbery which will work alongside the current gun computer. This one will spawn more crates with more/better guns but it's a much higher risk with PD and other crims. I think it could be a nice option though.

**Comment by coalaura** [comment]
Due to the recent changes and the state the gun market is in rn i'm going to close this issue for now if it happens to be a  problem in the future we can reopen it again or make a new one
